cabal-install/src/Distribution/Client/CmdInstall.hs

```diff
@@ -711,6 +722,10 @@ installLibraries verbosity buildCtx compiler
         ++ "so only executables will be available. (Library installation is "
         ++ "supported on GHC 8.0+ only)"
 
+
+globalPackages :: [PackageName]
+globalPackages = mkPackageName <$> [ "base" ]
```

Comment:
I think we should add here a reason why we unconditionally add "base".

Reply:
And not any other built-in package or whatever the smallest enclosing set is.

Reply:
follow-up in https://github.com/haskell/cabal/issues/8938